app/assets/stylesheets/application.scss

```
@@ -85,4 +85,12 @@ padding-right: 8px;
     margin: 0 auto;
   }
   
-}
\ No newline at end of file
+  .main{
```

タグを指定する時に`.`を入れてはいけません。
```suggestion
  main{
```